exercises/luhn-trait/HINTS.md

```diff
@@ -0,0 +1,16 @@
+# Luhn Validation with a Trait
+
+Before doing this exercise you should probably do the original Luhn exercise and its successor, Luhn From Trait 
+
+- `exercism fetch rust luhn`
+- `exercism fetch rust luhn-from`
+
+In the original Luhn exercise you only validated strings, but the Luhn algorithm can be applied to integers as well.
+
+In Luhn From you implemented a From trait, which also required you to create a Luhn struct.
```

Comment:
here, it says "Luhn From", whereas above it says "Luhn From Trait". In the header of the `HINTS.md` of the `luhn-from`, it says "Luhn with From Trait". Intentional? Seems OK I guess since they should all be distinguishable.